powerdnsadmin/routes/admin.py

```diff
@@ -93,7 +93,7 @@ def extract_changelogs_from_a_history_entry(out_changes, history_entry, change_n
         return
 
     if "add_rrests" in history_entry.detail:
-        detail_dict = json.loads(history_entry.detail)
+        detail_dict = json.loads(history_entry.detail.replace("\'", ''))
```

Comment:
@vmarkop @kkmanos You worked on detail history, can you remind us why we added this replacement ?

Thanks

Reply:
Hello,

the above command removes any single quotes that were escaped with a backslash, as their existence used to break history rendering and throw errors.

Thank you

Reply:
@jbe-dw I cant reproduce that issue on my local environment. How can I cause it?

Reply:
Hello,

Because before your patch history details are written with single quotes. json.loads crashes because it is not a valid JSON.

You can try to update one of the 4 most recent history entry (like successfull login) in your db then reload the dashboard.

Reply:
@vmarkop 

I have worked on the Alembic script pushed in this PR. It should solves the corrupted quote issues fixed L766.

For line 96, I need some corrupted history entries addressed by this interpolation to produce a clean patch as well.

Considering that it was added by your team, I really rely on you to gather it (either from one of your managed instance or maybe a ticket open by your customer).

I think it is far better to fix the db log rather than patching the python code. As @AdvanticGmbH, we should store valid JSON as we can in text columns.

If you can take some time to test the migration script on your side it would be appreciated.